test/test_ui.py

```diff
@@ -934,10 +934,12 @@ def test_default_paths_preserved(self):
 
     def test_nonexistant_db(self):
         with self.write_config_file() as config:
-            config.write("library: /xxx/yyy/not/a/real/path")
+            with control_stdin("y"):
+                config.write("library: /xxx/yyy/not/a/real/path")
```

Comment:
Is this actually doing anything? From quick glance at the code, there should be no way this could prompt for user input.

Reply:
Right; I think we probably don't need the `control_stdin` wrapper here.

Reply:
I accidentally globbered by old commit and history with a botched `git push -f`, so I can't see what I had before, but this is what it **should** have been:

```python
with self.write_config_file() as config:
    config.write("library: /xxx/yyy/not/a/real/path")

with self.assertRaises(ui.UserError):
    with control_stdin("n"):
        self.run_command("test", lib=None)
```

Without it, I don't see anything globally set that's mocking stdin.
With `control_stdin` here, the call the call to `input` ([here](https://github.com/beetbox/beets/blob/master/beets/ui/__init__.py#L202)) is handled w/o need for user input, thereby making the test pass.

I'm not sure how this test was passing before **without** this?

I re-added it and it's passing now.
Just to make sure, I rebased on commit `c1a232ec` of `master`, and it's working.

Let me push this change again, this time correctly 🙂